packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1778,3 +1737,24 @@ enum NonNullSafeBuilds {
   allowed,
   notAllowed,
 }
+
+extension on WebRendererMode {
+  Iterable<String> get dartDefines => switch (this) {
```

Comment:
We *could* embed this in the enum itself, but then we're leaking things across domains. 🤷

Reply:
I like this as is, but does it need to be an `extension`? Why not make it part of the enhanced enum outright?

Reply:
I could...